content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -77,10 +77,10 @@ ssh -D 1080 -q -N username@kubernetes-remote-server.example
 To explore the Kubernetes API you'll first need to instruct your clients to send their queries through
 the SOCKS5 proxy we created earlier.
 
-For command-line tools, set the `https_proxy` environment variable and pass it to commands that you run.
+For command-line tools, set the `https_proxy` environment variable and pass it to commands that you run (on some cases it may be `HTTPS_PROXY`).
 
 ```shell
-export https_proxy=socks5h://localhost:1080
+export https_proxy=socks5://localhost:1080
```

Comment:
No. This is not a typo... There had been several attempts to "fix" this, because people doesn't know "socks5h://".

Reply:
You use socks5h only if there is concern for DNS leakage. Certainly not in the case of localhost! So its use is for very specific situation not relevant here, and `socks5` will always work, whereas not every program supports `socks5h`. Plus few people know about it, and anyone who does know will also know that it can be used. See https://superuser.com/a/1762355/199881 for details. 

If mulitple people have suggested a fix, then it should be taken as a clear sign that the documentation needs fixing. 

So let's make everyone's life easier by removing it (both readers and doc maintainers will appreciate the reduction in noise).

Reply:
For quite some users (especially those behind a damn national firewall), proxying DNS query is a must.
If we instead clarify the difference between `socks5://` and `socks5h://` rather than treating `socks5h` as a typo, I'd appreciate such a PR.